src/server/test/db/common.js

```diff
@@ -16,16 +16,23 @@ config.database = {
 	port: process.env.OED_DB_TEST_PORT || process.env.OED_DB_PORT
 };
 
+const { getDB, currentDB, createSchema } = require('../../models/database');
+
 // Disable logging during tests.
 // TODO: Move logging disabling to a better place.
 log.level = LogLevel.SILENT;
 
-const { db, createSchema } = require('../../models/database');
-
 async function recreateDB() {
+	// Just transfer connection if needed.
+	getDB();
+	if (currentDB() === process.env.OED_DB_DATABASE) {
+		// tslint:disable-next-line no-console
+		console.error(`Refusing to destroy production database ${currentDB()} for tests; should be using ${config.database.database}.`)
```

Comment:
The Travis test is returning an error from this line showing "Refusing to destroy production database travis_ci_test for tests; should be using travis_ci_test." config.database.database refers to process.env.OED_DB_DATABASE in src/server/config.js and they are the same. I guess either the if statement or the console error message is mistaken.